base/utferror.jl

```diff
@@ -0,0 +1,30 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+##\brief      Error messages for Unicode / UTF support
```

Comment:
And I was just going to change this to using something more along the lines of Mike's idea... :grinning:  I guess I can do that in a later PR, as soon as the dust as settled a bit on the parsing Markdown ideas...  Thanks!  (and thanks for taking the time, *and* pushing back on the doxygen format, because now I will push for something better for documentation, which will still allow me to generate output for doxygen, but which fits better with Julia, and which I hope you *do* like)